MobileWallet/Backup/ICloudBackup.swift

```diff
@@ -580,3 +606,17 @@ extension ICloudBackup {
         try FileManager.default.removeItem(at: directory)
     }
 }
+
+// MARK: Tests
+extension ICloudBackup {
+    private func isTest() -> Bool {
```

Comment:
You can use an existing flag I have in the settings rather. `TariSettings.shared.isUnitTesting`